src/components/EnvelopeList.vue

```diff
@@ -272,6 +273,10 @@ export default {
 		selectedEnvelopes() {
 			return this.envelopes.filter((env) => this.selection.includes(env.databaseId))
 		},
+		hasMultipleAccounts() {
+			const mailboxIds = this.envelopes.map(envelope => envelope.mailboxId)
+			return Array.from(new Set(mailboxIds)).length > 1
```

Comment:
It would hurt to comment such code. I get that the conversion from array to set to array is to remove duplicates but it's not obvious :)